torch/_export/converter.py

```diff
@@ -340,8 +350,10 @@ def get_fx_value(self, value: torch._C.Value):
         if value_name in self.name_to_node:
             input_node = self.name_to_node[value_name]
             return input_node
-        elif value_name in self.constant_map:
-            return self.constant_map[value_name]
+        elif value_name in self.name_to_constant:
+            if isinstance(self.name_to_constant[value_name], torch.ScriptObject):
+                return self.fx_graph.get_attr(value_name)
```

Comment:
do we need to manually create the getattr node? Does `return self.constant_map[value_name]` not just automatically create the getattr node?

Reply:
Right, I think we discussed that we want to only keep the actual value in the `name_to_constant` map. And every time any nodes tries to use the actual value, we insert an ad-hoc `getattr`.